app/view/main/Main.js

```diff
@@ -44,5 +45,7 @@ Ext.define('CpsiMapview.view.main.Main', {
         }
     }, {
         xtype: 'cmv_header'
+    }, {
+        xtype: 'cmv_line_slice_grid_window'
```

Comment:
How about instantiating the `cmv_line_slice_grid_window` in the handler of `app/view/lineSliceGridExample/LineSliceGridButton.js` instead of hanging it in the main view?